__tests__/__system__/cli/delete/urimap/cli.delete.urimap.system.test.ts

```diff
@@ -0,0 +1,267 @@
+            [urimapName, csdGroup, regionName]);
+        stderr = output.stderr.toString();
+        expect(stderr).toEqual("");
+        expect(output.status).toEqual(0);
+        expect(output.stdout.toString()).toContain("success");
+    });
+
+    it("should be able to delete a urimap of type pipeline with basic options", async () => {
+        const urimapNameSuffixLength = 3;
+        const urimapName = "X" + generateRandomAlphaNumericString(urimapNameSuffixLength);
+        const urimapPath = "fake/path";
+        const urimapHost = "www.example.com";
+        const pipelineName = "FAKEPIPE";
+
+        let output = runCliScript(__dirname + "/../../define/urimap-pipeline/__scripts__/define_urimap_pipeline.sh", TEST_ENVIRONMENT,
+            [urimapName, csdGroup, urimapPath, urimapHost, pipelineName, regionName]);
+        let stderr = output.stderr.toString();
+        expect(stderr).toEqual("");
+        expect(output.status).toEqual(0);
+        expect(output.stdout.toString()).toContain("success");
+
+        output = runCliScript(__dirname + "/__scripts__/delete_urimap.sh", TEST_ENVIRONMENT,
+            [urimapName, csdGroup, regionName]);
+        stderr = output.stderr.toString();
+        expect(stderr).toEqual("");
+        expect(output.status).toEqual(0);
+        expect(output.stdout.toString()).toContain("success");
+    });
+
+    it("should be able to delete a urimap of type client with basic options", async () => {
+        const urimapNameSuffixLength = 3;
+        const urimapName = "X" + generateRandomAlphaNumericString(urimapNameSuffixLength);
+        const urimapPath = "fake/path";
+        const urimapHost = "www.example.com";
+
+        let output = runCliScript(__dirname + "/../../define/urimap-client/__scripts__/define_urimap_client.sh", TEST_ENVIRONMENT,
+            [urimapName, csdGroup, urimapPath, urimapHost, regionName]);
+        let stderr = output.stderr.toString();
+        expect(stderr).toEqual("");
+        expect(output.status).toEqual(0);
+        expect(output.stdout.toString()).toContain("success");
+
+        output = runCliScript(__dirname + "/__scripts__/delete_urimap.sh", TEST_ENVIRONMENT,
+            [urimapName, csdGroup, regionName]);
+        stderr = output.stderr.toString();
+        expect(stderr).toEqual("");
+        expect(output.status).toEqual(0);
+        expect(output.stdout.toString()).toContain("success");
+    });
+
+    it("should get a syntax error if csdGroup is omitted", () => {
+        const output = runCliScript(__dirname + "/__scripts__/delete_urimap.sh", TEST_ENVIRONMENT, ["FAKENAME", ""]);
+        const stderr = output.stderr.toString();
+        expect(stderr).toContain("Syntax");
+        expect(stderr).toContain("Missing Positional Argument");
+        expect(stderr).toContain("csdGroup");
+        expect(output.status).toEqual(1);
+    });
+
+    it("should get a syntax error if csdGroup is omitted", () => {
```

Comment:
Shouldn't this say `if urimap is omitted`?

Reply:
Yep, good catch. Thanks.